src/style.css

```
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  font-family: inherit;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__logo {
+  display: flex;
+  background-repeat: no-repeat;
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__text {
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px 16px 4px;
+  color: #060b35;
+}
+
+.card__second-text {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0;
+  padding-left: 16px;
+  padding-bottom: 16px;
+  color: #616070;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+ 
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:not(:last-child) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__reviews {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
```

```suggestion
  font-family: inherit;
  font-weight: 400;
```
its default, remove it